lib/health.js

```diff
@@ -7,8 +7,7 @@ const STATUS_WARN = 'warn';
 
 module.exports = function(opts) {
 
-  const healthURI = require('config').uri;
-
+  const healthURI = process.env.NODE_HEALTH_ENDPOINT_PATH || "/health";
```

Comment:
Because this is a module that is supposed to be used from an app and none of the config magic was working in that scenario. Bottomline: `config` should only be used from the top-level app, never in the modules

Reply:
...which raises the question: why have the config directory at all?

Reply:
no reason, just something forgotten https://github.com/inadarei/maikai/pull/5